doc/release-notes/5.10-release-notes.md

```diff
@@ -0,0 +1,315 @@
+
+This release brings new features, enhancements, and bug fixes to the Dataverse Software. Thank you to all of the community members who contributed code, suggestions, bug reports, and other assistance across the project.
+
+## Release Highlights
+
+### Multiple License Support
+
+Users can now select from a set of configured licenses in addition to or instead of the previous Creative Commons CC0 choice or provide custom terms of use (if configured) for their datasets. Administrators can configure their Dataverse instance via API to allow any desired license as a choice and can enable or disable the option to allow custom terms. Administrators can also mark licenses as "inactive" to disallow future use while keeping that license for existing datasets. For upgrades, only the CC0 license will be preinstalled. New installations will have both CC0 and CC BY preinstalled. The [Configuring Licenses](https://guides.dataverse.org/en/5.10/installation/config.html#configuring-licenses) section of the Installation Guide shows how to add or remove licenses.
+
+**Note: Datasets in existing installations will automatically be updated to conform to new requirements that custom terms cannot be used with a standard license and that custom terms cannot be empty. Administrators may wish to manually update datasets with these conditions if they do not like the automated migration choices. See the "Notes for Dataverse Installation Administrators" section below for details.**
+
+This release also makes the license selection and/or custom terms more prominent when publishing and viewing a dataset and when downloading files.
+
+### Ingest and File Upload Messaging Improvements
+
+Messaging around ingest failure has been softened to prevent support tickets. In addition, messaging during file upload has been improved, especially with regard to showing size limits and providing links to the guides about tabular ingest. For screenshots and additional details see PR #8271.
+
+### Downloading of Guestbook Responses with Fewer Clicks
+
+A download button has been added to the page that lists guestbooks. This saves a click but you can still download responses from the "View Responses" page, as before.
+
+Also, links to the guides about guestbooks have been added in additional places.
+
+### Dynamically Request Arbitrary Metadata Fields from Search API 
+
+The Search API now allows arbitrary metadata fields to be requested when displaying results from datasets. You can request all fields from metadata blocks or pick and choose certain fields.
+
+The new parameter is called `metadata_fields` and the Search API documentation contains details and examples: <https://guides.dataverse.org/en/5.10/api/search.html>
+
+### Solr 8 Upgrade
+
+The Dataverse Software now runs on Solr 8.11.1, the latest available stable release in the Solr 8.x series.
+
+### PostgreSQL Upgrade
+
+A PostgreSQL upgrade is not required for this release but will be in the future. See below for details.
```

Comment:
should we just say "... will be in the next release"?

Reply:
@landreev good idea. In ce26e60fbb I indicated the next release but hedged a bit by saying "planned for the next release".